File: Code/Client/src/app/models/Notificacion.ts
export class Notificacion {

    Id: string;
    CarnetUsuario: string | null;
    Titulo: string | null;
    Contenido: string | null;
    FechaEnvio: string | null;
    Leido : boolean | null;

    constructor() {
        this.Id = '';
        this.CarnetUsuario = null;
        this.Titulo = null;
        this.Contenido = null;
        this.FechaEnvio = null;
        this.Leido = null;
    }

}

File: Code/Server/src/Core/Entities/Notificacion.cs
namespace IMT_Reservas.Server.Core.Entities;

public class Notificacion : Entity
{
    public int IdUsuario { get; set; }
    public string? Titulo { get; set; }
    public string? Contenido { get; set; }
    public bool EsLeido { get; set; }
    public DateTime FechaCreacion { get; set; }
    public bool EstadoEliminado { get; set; }
}
